docs/download_files.md

```diff
@@ -139,6 +139,45 @@ For more on the specific contents of multiplexed library `SingleCellExperiment`
+
+Merged object downloads contain all single-cell or single-nuclei gene expression data for a given ScPCA project within a single object, provided as either a [`SingleCellExperiment` object (`.rds` file)](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html) or an [`AnnData` object (`.hdf5` file)](https://anndata.readthedocs.io/en/latest/index.html).
+
+The object file, `SCPCP000000_merged.rds` or `SCPCP000000_merged_rna.hdf5`, contains both a raw and normalized counts matrix, each with combined counts for all samples in an ScPCA project.
+In addition to the counts matrices, the `SingleCellExperiment` or `AnnData` object stored in the file includes the results of library-weighted dimensionality reduction using both principal component analysis (PCA) and UMAP.
+See the {ref}`section on merged object processing<processing_information:merged objects>` for more information about how merged objects were created.
+
+If downloading a project that contains at least one CITE-seq library, the quantified CITE-seq expression data will also be merged.
+In `SingleCellExperiment` objects (`rds` files), the  CITE-seq expression data is be provided as an alternative experiment in the same object as the gene expression data.
+However, for `AnnData` objects, (`hdf5` files), the quantified CITE-seq expression is instead provided as a separate file called `SCPCP000000_merged_adt.hdf5`.
+
+Every download includes a summary report, `SCPCL000000_merged-summary-report.html`, which provides a brief summary of the samples and libraries included in the merged object.
+This includes a summary of the types of libraries (e.g., single-cell, single-nuclei, with CITE-seq) and sample diagnoses included in the object, as well as UMAP visualizations highlighting each library.
+
+Every download also includes the individual [QC report](#qc-report) and, if applicable, [cell type annotation reports](#cell-type-report) for each library included in the merged object.
+
+### Download folder structure for `SingleCellExperiment` merged downloads:
+_image pending_
+
+### Download folder structure for `AnnData` merged downloads:
+_image pending_
+
+### Download folder structure for `AnnData` merged downloads with CITE-seq (ADT) data:
+_image pending_
+
+
+### Merged object metadata
+
+Similar to downloading the project with individual files for each sample, downloading the project as a merged object includes a single `single_cell_metadata.tsv` file containing metadata for all libraries included in the download.
+
+If downloading a project containing bulk RNA-seq data, two tab-separated value files, `bulk_quant.tsv` and `bulk_metadata.tsv`, will be included in the project download.
+The `bulk_quant.tsv` file contains a gene by sample matrix (each row a gene, each column a sample) containing raw gene expression counts quantified by Salmon.
+The `bulk_metadata.tsv` file contains associated metadata for all samples with bulk RNA-seq data.
```

Comment:
I think I might move this section to be right below the images or right above. Putting it right below would mirror how we mention bulk for the single sample downloads.

Reply:
Do you mean the bulk text specifically or the little section for merged metadata downloads? Currently this is right below the images.
![Screenshot 2024-01-22 at 1 48 37 PM](https://github.com/AlexsLemonade/scpca-docs/assets/4701111/be9357e4-e999-4c7c-9fa0-a9ece4a26ea5)

Reply:
I mean to move the `If downloading a project containing bulk RNA-seq` and the three lines that mention bulk to be in the section that actually shows the download images and remove from the header `Merged object metadata`.